src/components/AccountsManagementComponent.jsx

```diff
@@ -548,7 +552,7 @@ class AccountsManagementComponent extends Component {
       >
         <Container className={classes.container}>
           <Grid container>
-            <Grid item md={6} xs={12} className={classes.upperBigGridItem}>
+            <Grid item lg={6} xs={12} className={classes.upperBigGridItem}>
```

Comment:
why change this?

Reply:
Added a new btn called 'Add New Token' here, have to change the breakpoint to be at larger screen size, otherwise break too soon.